DistroLauncher/DistroLauncher.cpp

```diff
@@ -38,8 +38,9 @@ HRESULT InstallDistribution(bool createUser, Oobe::Application<>& app)
     }
 
     // Prepare distro for systemd enablement, and conditionally enable it
-    const bool enable_systemd =
-      ends_with(DistributionInfo::Name, L".Dev") || DistributionInfo::Name == L"Ubuntu-Preview";
+    const bool enable_systemd = ends_with(DistributionInfo::Name, L".Dev") ||
+      DistributionInfo::Name == L"Ubuntu" || DistributionInfo::Name == L"Ubuntu-22.04" ||
+      DistributionInfo::Name == L"Ubuntu-Preview";

```

Comment:
At this point we may want to black-list 18.04 and 20.04 rather than white-list all others 😅.

Reply:
In the coming weeks, when all the installed services are fixed, we'll enable it for all the releases. So this condition won't be necessary anymore.